stable/syncthing/SCALE/questions.yaml

```diff
@@ -141,17 +141,120 @@ questions:
+                        required: true
+        - variable: listeners
+          label: "Syncthing listening ports"
+          description: "This service is used to process incoming connections directly to this Syncthing instance"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable the service"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: false
+              - variable: type
+                label: "Service type"
+                description: "ClusterIP's are only internally available, nodePorts expose the container to the host node System"
+                schema:
+                  type: string
+                  default: "NodePort"
+                  enum:
+                    - value: "NodePort"
+                      description: "NodePort"
+                    - value: "ClusterIP"
+                      description: "ClusterIP"
+              - variable: port
+                label: "TCP port"
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: protocol
+                      label: "Port Type"
+                      schema:
+                        type: string
+                        default: "UDP"
+                        hidden: true
+                        enum:
+                          - value: TCP
+                            description: "TCP"
+                          - value: "UDP"
+                            description: "UDP"
+                    - variable: port
+                      label: "Container port"
+                      schema:
+                        type: int
+                        default: 22000
+                        editable: true
+                        hidden: false
+                    - variable: targetport
+                      label: "Internal Service port"
+                      description: "When connecting internally to this App, you'll need this port"
+                      schema:
+                        type: int
+                        default: 22000
+                        editable: true
+                        hidden: false
+                    - variable: nodePort
+                      label: "(optional) host nodePort to expose to"
+                      description: "only get used when nodePort is selected"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 22000
+                        required: true
+              - variable: port
+                label: "UDP port"
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: protocol
+                      label: "Port Type"
+                      schema:
+                        type: string
+                        default: "TCP"
```

Comment:
I don't think syncthing uses both tcp and udp, or does it?

Reply:
Apparently it uses tcp and quic, their own docker image example tells you to use both TCP and UDP.

Reply:
Okey, i'll consider that resolved.